packages/fxa-auth-server/lib/senders/emails/templates/verifyShortCode/en.ftl

```diff
@@ -2,7 +2,10 @@
 #  $code (Number) - e.g. 123456
 verifyShortCode-subject-3 = Confirm your account
 verifyShortCode-title-2 = Open the internet with { -brand-firefox }
+verifyShortCode-title-3 = Open the internet with { -brand-mozilla }
```

Comment:
Was this a string you double checked with UX? I can see this being debatable. I think it makes sense, but just double checking here.

Reply:
Just posted a question in the slack channel, I had not confirmed this one.